src/platform/testing/unit/mocha.opts

```diff
@@ -8,4 +8,4 @@
 --compilers js:@babel/register,jsx:@babel/register
 --prof
 --reporter progress
---timeout 5000
+--timeout 10000
```

Comment:
Since the NPM script for contract tests uses the general unit test script now, I also increased the general timeout to accommodate that.